lang/user/id.xml

```diff
@@ -332,31 +327,31 @@ Sambungkan Kontroleryang kompatibel dengan SDL2.</not_connected>
 		</trophy>
 		<save_data>
 			<delete>
-				<cancel_deleting>Kamu Yakin untuk batalkan penghapusan ?</cancel_deleting>
+				<cancel_deleting>Kamu yakin untuk batalkan penghapusan?</cancel_deleting>
 				<deletion_complete>Penghapusan Berhasil.</deletion_complete>
-				<delete_saved_data>Kamu yakin untuk hapus  save data ini?</delete_saved_data>
+				<delete_saved_data>Kamu yakin untuk hapus save data ini?</delete_saved_data>
 			</delete>
 			<info>
 				<details>Detail</details>
 				<updated>Update telah selesai</updated>
 			</info>
-			<load name="Load">
+			<load name="Tunggu">
 				<cancel_loading>Kamu yakin untuk batalkan loading?</cancel_loading>
 				<no_saved_data>Disini gak ada save data.</no_saved_data>
 				<load_complete>Loading Berhasil.</load_complete>
-				<loading>Loading...</loading>
+				<loading>Tunggu...</loading>
 				<load_saved_data>Kamu yakin untuk load di save data ini?</load_saved_data>
 			</load>
-			<save name="Save">
+			<save name="Nyimpan">
```

Comment:
Menyimpan

Reply:
Nyimpan Just it so it doesn't become baku